cmd/bucket-targets.go

```diff
@@ -351,6 +351,25 @@ func (sys *BucketTargetSys) SetTarget(ctx context.Context, bucket string, tgt *m
 			return BucketRemoteTargetNotVersioned{Bucket: tgt.TargetBucket}
 		}
 	}
+
+	// Check if target is a MinIO server and alive
+	hcCtx, cancel := context.WithTimeout(ctx, 30*time.Second)
```

Comment:
This is too long, keep it like 3 secs, it is more than enough. Anywhere where latency is bigger than 3 seconds then it's a unusable cluster anyway.

We can even indicate to the user that their remote target doesn't respond to the expected 3 sec response time.

This way they know what is expected from a replication target.